src/index.js

```diff
@@ -71,158 +71,188 @@ class App extends React.Component {
+        <span>
+          Demo page for <mark>React Transcript Editor</mark> - Component |{' '}
+          <a
+            href="https://github.com/bbc/react-transcript-editor"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
             Github Repo
-           </a>
-         </span>
-         <br />
-         <button onClick={ () => this.loadDemo() }>load demo</button>
-         <hr />
-         <label>Load Local Media</label>
-         <input
-           type="file"
-           onChange={ e => this.handleChangeLoadMedia(e.target.files) }
-         />
-         or
-         <button onClick={ () => this.handleChangeLoadMediaUrl() }>
-          Load Media From Url
-         </button>
-         <br/>
-         <label>open Transcript Json</label>
-         <SttTypeSelect
-           name={ 'sttType' }
-           value={ this.state.sttType }
-           handleChange={ this.handleSttTypeChange }
-         />
-         <input
-           type="file"
-           onChange={ e => this.handleChangeLoadTranscriptJson(e.target.files) }
-         />
-
-         <br />
-         <label>Export transcript</label>
-         <button onClick={ () => this.exportTranscript() }>Export file</button>
-
-         <ExportFormatSelect
-           name={ 'exportFormat' }
-           value={ this.state.exportFormat }
-           handleChange={ this.handleExportFormatChange }
-         />
-         <br />
-         <label>Text Is Editable</label>
-         <label>
-           <input type="checkbox"
-             defaultChecked="true"
-             onChange={ this.handleIsTextEditable }
-           />
-         </label>
-         <br/>
-         <label>Optional Transcript Name</label>
-         <input
-           type="text"
-           value={ this.state.title }
-           onChange={ e => this.handleChangeTranscriptTitle(e.target.value) }
-         />
-
-         <br />
-         <button onClick={ () => this.clearLocalStorage() }>Clear Local Storage</button>
-         <hr/>
-
-         <TranscriptEditor
-           transcriptData={ this.state.transcriptData }
-           fileName={ this.state.fileName }
-           mediaUrl={ this.state.mediaUrl }
-           isEditable={ this.state.isTextEditable }
-           sttJsonType={ this.state.sttType }
-           handleAnalyticsEvents={ this.handleAnalyticsEvents }
-           title={ this.state.title }
-           ref={ this.transcriptEditorRef }
-         />
-         <hr/>
-         <label>Components Analytics</label>
-         <textarea
-           style={ { height: '200px', width: '100%' } }
-           value={ JSON.stringify(this.state.analyticsEvents, null, 2) }
-           disabled>
-         </textarea>
-       </div>
-     );
-   }
+          </a>
+          <hr />
+        </span>
+
+        <div className={ style.demoNav }>
+          <section className={ style.demoNavItem }>
+            <button onClick={ () => this.loadDemo() }>load demo</button>
```

Comment:
Some consistency with text formatting would make this all look cleaner.

`load demo`
`Load Local Media`
`open Transcript`
`Export transcript`

the capitalisation is all over the place and makes it look a lil messy